bot/exts/code_jams/_cog.py

```diff
@@ -0,0 +1,236 @@
+import asyncio
+import csv
+import typing as t
+from collections import defaultdict
+
+import discord
+from botcore.utils.logging import get_logger
+from botcore.utils.members import get_or_fetch_member
+from discord import Colour, Embed, Guild, Member
+from discord.ext import commands
+
+from bot.bot import SirRobin
+from bot.constants import Emojis, Roles
+from bot.exts.code_jams import _channels
+from bot.services import send_to_paste_service
+
+log = get_logger(__name__)
+
+TEAM_LEADERS_COLOUR = 0x11806a
+DELETION_REACTION = "\U0001f4a5"
+
+
+class CodeJams(commands.Cog):
```

Comment:
Some of the commands in this cog have required arguments, which result in `discord.ext.commands.errors.MissingRequiredArgument` being raised if they are given. This should be handled either with a cog-specific error handler, or a generic error handler cog.